SSA/Projects/InstCombine/AliveStatements.lean

```diff
@@ -9,8 +10,11 @@ open Std.BitVec
 open LLVM
 
 @[simp] lemma getLsb_negOne' (w : ℕ) (i : Fin w) :
-    getLsb (-1#w) ↑i := by 
-  simp only [getLsb_val_eq_getLsb', getLsb'_neg_ofNat_one]
+    getLsb (-1#w) i := by
```

Comment:
I'm confused from seeing changes in this file. This is an auto-generated file, I thought we were not going to manually change this file anymore.

Reply:
@goens, what is the process to get this file updated properly? Happy to follow your suggestions here.

Reply:
that's a good question, I don't know if we've refined this properly, we talked about it before the rebuttal, maybe @bollu remembers what the final process was? Ideally, these changes would go into the `alive_auto` tactic or the `extract_alive_statements.sh` script.

Reply:
Right. Maybe document the processes here among the two of you. I can then follow this process to regenerate this file.